doc/entity.md

```diff
@@ -0,0 +1,117 @@
+# Entity
+
+An entity is the way you describe objects you want to work with in wetland, wetland will create the sql schema for you.
+
+## Entity example :
+
+Here's a user with an id in autoincrement mode, a username and a password.
+
+```js
+const {Entity} = require('wetland');
```

Comment:
I think maybe remove this const. It's only needed to extend Entity if you wish to use the helpers (`toObject() for instance`).